mdtraj/geometry/rdf.py

```diff
@@ -158,21 +160,30 @@ def compute_rdf_t(traj, pairs, times, period_length=None, r_range=None,
         pairs_set = np.unique(pairs)
         pairs = np.vstack([np.vstack([pairs_set, pairs_set]).T, pairs])
 
-    g_r_t = np.zeros(shape=(len(times), n_bins))
-    num_chunks = int(np.floor(traj.n_frames / period_length))
-
-    # Returns shape (len(times), len(pairs))
-    frame_distances = compute_distances_t(traj, pairs, times, periodic=periodic, opt=opt)
-
-    for n, distances in enumerate(frame_distances):
-        tmp, edges = np.histogram(distances, range=r_range, bins=n_bins)
-        g_r_t[n, :] += tmp
-    r = 0.5 * (edges[1:] + edges[:-1])
-
-    # Normalize by volume of the spherical shell (see above)
-    V = (4 / 3) * np.pi * (np.power(edges[1:], 3) - np.power(edges[:-1], 3))
-    norm = len(pairs) / (period_length) * np.sum(1.0 / traj.unitcell_volumes) * V
-
-    g_r_t = g_r_t.astype(np.float64) / norm  # From int64.
-
-    return r, g_r_t
+    g_r_t = []
+    weights = []
```

Comment:
I always thought initializing an empty array was better practice over starting with an empty list, but I never looked into why, or if that was even the case. It doesn't look like this is a major part of this patch, so it's more a point of curiosity as long as it runs well and does what you and @rmatsum836 need it to d.

Reply:
@jacoblumpkins Is there a reason why you changed `type(g_r_t)` from array to list?  

Initializing an empty NumPy array should be more performant than initializing an empty list (specifying exact memory vs overallocting memory for a list and copying objects over to a new list when memory is full) so we should try to stay with NumPy if possible.

Reply:
I just made the array type changes and added a test for `n_concurrent pairs`.